assets/js/base/components/reviews/types.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { EmptyObjectType } from '@woocommerce/types';
+
+export type review =
```

Comment:
```suggestion
export type Review =
```

It is a good practice that the types start with capital letters.